dcc-portal-ui/app/scripts/pathwayviewer/views/viewer.html

```diff
@@ -24,11 +24,29 @@
       
       <table class="table pathway-gene-table" data-ng-if="annotatedGenesList.length>0">
         <tr>
-          <th class="pathway-gene-header-label pathway-gene-header">Gene</th>
-          <th class="pathway-gene-header-label pathway-gene-header"># ICGC Mutations</th>
-          <th class="pathway-gene-header-label pathway-gene-header"># Targeting Drugs</th>
+          <th class="pathway-gene-header-label pathway-gene-header">
+            <a href="" data-ng-click="sortField = 'symbol'; sortDescendingly = !sortDescendingly">
```

Comment:
I stand corrected, looks like it is a word!
http://www.merriam-webster.com/dictionary/descendingly